modules/ingest-common/src/main/java/org/opensearch/ingest/common/DotExpanderProcessor.java

```diff
@@ -118,25 +118,15 @@ public Processor create(
         ) throws Exception {
             String field = ConfigurationUtils.readStringProperty(TYPE, tag, config, "field");
             if (field.contains(".") == false) {
-                throw ConfigurationUtils.newConfigurationException(
-                    ConfigurationUtils.TAG_KEY,
-                    tag,
-                    "field",
-                    "field does not contain a dot"
-                );
+                throw ConfigurationUtils.newConfigurationException(TYPE, tag, "field", "field does not contain a dot");
```

Comment:
That makes sense, since we are reading `TYPE` as well (`ConfigurationUtils.readStringProperty(TYPE, tag, config,...`), thanks @gaobinlong